ktor-utils/jvm/src/io/ktor/util/reflect/TypeInfoJvm.kt

```diff
@@ -4,22 +4,15 @@
 
 package io.ktor.util.reflect
 
-import java.lang.reflect.*
-import java.lang.reflect.Type
 import kotlin.reflect.*
 
 public actual typealias Type = java.lang.reflect.Type
 
-@PublishedApi
-internal open class TypeBase<T>
```

Comment:
Since this class is annotated with `@PublishedApi`, it's part of the public API and can't be removed in minor release. Please keep it, but add a deprecation to it.